Source/DataSources/CzmlDataSource.js

```diff
@@ -1795,6 +1801,7 @@ define([
         processMaterialPacketData(polyline, 'material', polylineData.material, interval, sourceUri, entityCollection);
         processMaterialPacketData(polyline, 'depthFailMaterial', polylineData.depthFailMaterial, interval, sourceUri, entityCollection);
         processPacketData(Boolean, polyline, 'followSurface', polylineData.followSurface, interval, sourceUri, entityCollection);
```

Comment:
Are we deprecating `followSurface` from CZML?

Reply:
I think we should since `ArcType` completely supersedes it. Does CZML deprecation need to be specified anywhere?

Reply:
As far as I am aware, we have never broken CZML compatibility (after Cesium 1.0).  Can't we just implement `followSurface` in terms of `arcType` for backwards compatibility?

Reply:
Yes, that is what we are doing during the deprecation period. I can probably work it in.

Reply:
I added a mapping between follow-surface and `arcType` and added a comment to https://github.com/AnalyticalGraphicsInc/cesium/issues/7504 so that the spec is not broken once the option is removed from the other classes.